### ISSUE TITLE
Save the drawings in a proper backend

### ISSUE THREAD

**Issue by torwag** [JFK20/ha-draw#10]
If I get it right, the tldraw drawings are saved by now in the browser cache.
Users might want to have more than one canvas and access this via several devices.
As such, it might be nice if the tldraw canvas is saved properly on the ha server itself. 
Either in the database or as a file within the configuration  folder.

**Comment by JFK20** [comment]
no that not quite right. As you can check by drawing something and then pressing ctrl f5. Which reloads without cache. The drawing is still there. The Canvas is already stored to a Database via [tldraw Persistence Key](https://tldraw.dev/docs/persistence)

**Comment by torwag** [comment]
But it is save locally on the browser side, or not? 
I can give it a test. But reading the code I thought that the save location is locally on the client side.

**Comment by JFK20** [comment]
yeah right, It isnt stored in the browser cache but still in the Browser. But they luckely provide a way to store the data on the server.

**Comment by JFK20** [comment]
For the Persistence, one Problem, a rose  that it isn't possible to directly write to the file system from the card. A solution is a [custom Integration](https://github.com/JFK20/ha_draw_persistence) I created. It exposes an API Endpoint which sends and Receives Json which is stored in a file or is used to update the Canvas. It is needed to use the Persistence Buttons.
